docker/Dockerfile-prod

```diff
@@ -4,22 +4,59 @@
 
 FROM python:3.5-alpine
 
-RUN addgroup -g 1001 app && \
-    adduser -D -u 1001 -G app -s /usr/sbin/nologin app
+MAINTAINER mars@mozilla.com
+# These are unlikely to change from version to version of the container
+EXPOSE 9000
+CMD ["/usr/local/bin/uwsgi"]
+
+RUN addgroup -g 10001 app && adduser -D -u 10001 -G app -h /app app
+
+# uWSGI configuration
+ENV UWSGI_MODULE=landoapi.wsgi:app \
+	UWSGI_SOCKET=:9000 \
+	UWSGI_MASTER=1 \
+	UWSGI_WORKERS=2 \
```

Comment:
They can set these values by editing the Dockerfile, setting them in the Docker environment, or by passing in a uWSGI config file.  I'll let @ckolos decide which is best.  We can iterate on the Dockerfile to find the approach that works best for them.